jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -116,13 +180,18 @@ async def run(loop, application: Flask = None):  # pylint: disable=redefined-out
         application.logger.debug(f'enable-involuntary-dissolution flag on: {flag_on}')
         if flag_on:
             # check if batch can be run today
-            new_dissolutions_schedule_config = Configuration.find_by_name(config_name='DISSOLUTIONS_STAGE_1_SCHEDULE')
-            tz = pytz.timezone('US/Pacific')
-            cron_valid = croniter.match(new_dissolutions_schedule_config.val, tz.localize(datetime.today()))
-            if cron_valid:
+            cron_valids = check_run_schedule()
+            if not any(cron_valids):
+                application.logger.debug('Skipping job run since current day of the week does not match any cron schedule.')  # noqa: E501
+                return
+
+            if cron_valids[0]:
```

Comment:
Do you think it would make sense to return a `dict` from `check_run_schedule` instead of a list to improve readability? 

We could then check this as `cron_valids['stage_1']`. I'm not sure how we handle similar situations elsewhere in the code.

Otherwise I really like the new `check_run_schedule` function!

Reply:
My thoughts are maybe `check_run_schedule` is a little overloaded.

Maybe have a function that checks if a given cron string can run today.

And `check_run_schedule` can still check the 3 schedules.

To make things more readable as Leo suggested, maybe call `check_run_schedule` with `stage_1_valid, stage_2_valid, stage_3_valid = check_run_schedule()`.

We can also use a dataclass to return the validity of the stages.